src/sentry/api/endpoints/seer_rpc.py

```diff
@@ -459,6 +462,29 @@ def get_attributes_and_values(
     return {"attributes_and_values": attributes_and_values}
 
 
+def get_github_enterprise_integration_config(
+    *, organization_id: int, integration_id: int
+) -> dict[str, Any]:
+    integration = integration_service.get_integration(
+        integration_id=integration_id,
+        provider=IntegrationProviderSlug.GITHUB_ENTERPRISE.value,
+        organization_id=organization_id,
+        status=ObjectStatus.ACTIVE,
+    )
+    if integration is None:
+        raise Integration.DoesNotExist
+
+    installation = integration.get_installation(organization_id=organization_id)
+
+    return {
+        "base_url": f"https://{installation.model.metadata["domain_name"].split("/")[0]}/api/v3",
+        "private_key": installation.model.metadata["installation"]["private_key"],
```

Comment:
We'll want to encrypt this key prior to surfacing it to Seer via a shared secret (cc: @jennmueng @getsentry/security if you have preferred ways of doing this).